src/main/java/net/sf/jabref/logic/layout/format/RTFChars.java

```diff
@@ -206,46 +206,46 @@ private StringInt getPart(String text, int i, boolean commandNestedInBraces) {
      * @return returns the basic character of the given unicode
      */
     private String transformSpecialCharacter(long c) {
-        if((192 <= c && c <= 197) || c == 256 || c == 258 || c == 260) {
+        if(((192 <= c) && (c <= 197)) || (c == 256) || (c == 258) || (c == 260)) {
```

Comment:
Could you please run the formatter on the whole method (after using `gradlew eclipse` from master branch). Then, the necessary space after `if` should be added, too.